src/adUnit/helpers/vpaid/initAd.js

```diff
@@ -1,16 +1,34 @@
-import {getAdParameters} from '../../../vastSelectors';
+import {getCreativeData} from '../../../vastSelectors';
 import viewmode from './viewmode';
 
+const createSlot = (placeholder, width, height) => {
+  const slot = document.createElement('DIV');
+
+  slot.style.position = 'absolute';
```

Comment:
I find `Object.assign` in this case to be cleaner. I.e. 
```
Object.assign(slot.style, {
  position: 'absolute',
  top: '0px',
  ...
});
```

But up to you of course :)

Reply:
👍